test/src/main/java/io/strimzi/test/k8s/KubeClient.java

```diff
@@ -475,6 +475,14 @@ public Secret getSecret(String secretName) {
         return client.secrets().inNamespace(getNamespace()).withName(secretName).get();
     }
 
+    public Secret getSecret(String secretName, boolean namespaced) {
+        if (namespaced) {
+            return getSecret(secretName);
+        } else {
+            return client.secrets().withName(secretName).get();
+        }
+    }
+
```

Comment:
I don't really like the idea of putting boolean for this. Just create some separated method such as `getSecretFromAllNamespaces()` or whatever, where you'll return the `client.secrets().withName(asd).get();` instead of this.

Reply:
I will remove this method as well.